GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -369,7 +369,8 @@ private void SaveClick(object sender, EventArgs e)
 
                 if (!string.IsNullOrEmpty(result.UserMessage))
                 {
-                    MessageBox.Show(this, result.UserMessage, _gitMessage.Text);
+                    MessageBox.Show(this, result.UserMessage, _gitMessage.Text, MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
```

Comment:
This produces a nicer looking messagebox in my opinion, but we can revert if you like.